# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1787246942 -7200
#      Thu Aug 20 19:29:02 2026 +0200
# Branch stable
# Node ID cd0a12f887141deccf24d0832f6de923bd04ecbc
# Parent  8eff3586a5d5884bac43ed27f2efa95b92c5b809
build: Add missing linker flags for liboctinterp and liboctgui.

* configure.ac: Add BLAS_LIBS to LIBOCTINTERP_LINK_DEPS. Add OPENGL_LIBS to
LIBOCTGUI_LINK_DEPS.

See: https://octave.discourse.group/t/7693

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -3198,7 +3198,7 @@ LIBOCTAVE_LINK_OPTS="$CURL_LDFLAGS $SPAR
 AC_SUBST(LIBOCTAVE_LINK_DEPS)
 AC_SUBST(LIBOCTAVE_LINK_OPTS)
 
-LIBOCTINTERP_LINK_DEPS="$FT2_LIBS $HDF5_LIBS $MAGICK_LIBS $Z_LIBS $SPARSE_XLIBS $FFTW_XLIBS $OPENGL_LIBS $FONTCONFIG_LIBS $FREETYPE_LIBS $WAYLAND_CLIENT_LIBS $X11_LIBS $CARBON_LIBS $GL2PS_LIBS $JAVA_LIBS $LAPACK_LIBS"
+LIBOCTINTERP_LINK_DEPS="$FT2_LIBS $HDF5_LIBS $MAGICK_LIBS $Z_LIBS $SPARSE_XLIBS $FFTW_XLIBS $OPENGL_LIBS $FONTCONFIG_LIBS $FREETYPE_LIBS $WAYLAND_CLIENT_LIBS $X11_LIBS $CARBON_LIBS $GL2PS_LIBS $JAVA_LIBS $LAPACK_LIBS $BLAS_LIBS"
 
 LIBOCTINTERP_LINK_OPTS="$FT2_LDFLAGS $HDF5_LDFLAGS $MAGICK_LDFLAGS $WAYLAND_CLIENT_LDFLAGS $Z_LDFLAGS $SPARSE_XLDFLAGS $FFTW_XLDFLAGS"
 
@@ -3269,7 +3269,7 @@ OCTAVE_GUI_LINK_DEPS=""
 OCTAVE_GUI_LINK_OPTS=""
 
 if test $build_qt_gui = yes; then
-  LIBOCTGUI_LINK_DEPS="$QT_LIBS"
+  LIBOCTGUI_LINK_DEPS="$QT_LIBS $OPENGL_LIBS"
   LIBOCTGUI_LINK_OPTS="$QT_LDFLAGS"
 
   if test $link_all_deps = yes || test -n "$QT_LDFLAGS"; then
